### PR TITLE
doc: add more examples for mutation-test

### PR DESCRIPTION
**Pull request by Rqnsom**
Also includes a fix for `cargo install` command for `mutator-test`.

### CODE REVIEW

File: move-mutation-test/README.md [hunk 1]
```diff
@@ -148,4 +148,24 @@ Options:
   -h, --help                                           Print help (see more with '--help')
 ```
 
+### Examples
+
+_In below examples, the `RUST_LOG` flag is used to provide more informative output._
```
Comment (asmie) [inline move-mutation-test/README.md:153]:
```suggestion
_In below examples, the `RUST_LOG` flag is used to provide a more informative output._
```

File: move-mutation-test/README.md [hunk 2]
```diff
@@ -148,4 +148,24 @@ Options:
...
+To use the tool on only the `Operators` module for the project `simple`, run:
+```bash
+RUST_LOG=info ./target/release/move-mutation-test --package-dir move-mutator/tests/move-assets/simple -o report.txt --move-2 --mutate-modules Operators
+```
+------------------------------------------------------------------------------------------------------------
+To use the tool only on functions called `sum` for the project `simple`, run:
+```bash
+RUST_LOG=info ./target/release/move-mutation-test --package-dir move-mutator/tests/move-assets/simple -o report.txt --move-2 --mutate-functions sum
+```
+In the output for the above command, the tool will mutate both `Operators::sum` and `Sum::sum` functions.
```
Comment (asmie) [inline move-mutation-test/README.md:164]:
```suggestion
In the output for the above command, the tool will mutate both the `Operators::sum` and `Sum::sum` functions.
```

**Review by asmie — APPROVED** [review]
Nice! 👍

File: move-mutation-test/README.md (hunk 2 above)
Comment (Rqnsom) [reply move-mutation-test/README.md:164]: Done

File: move-mutation-test/README.md (hunk 1 above)
Comment (Rqnsom) [reply move-mutation-test/README.md:153]: Done

**Pull request merged**